### DISCUSSION TITLE
How to set user name on termux

### DISCUSSION THREAD

**Discussion by skydivingwill** [termux/termux-app#2696] [General]
I was looking for how to set username so I could use ssh to other servers with this app.
I saw several closed posts here that say it's not possible but it actually is.

- open Termux app
- create text file with: nano .ssh/config
- add the follow text:
Host *
    User YourUserName

ref: https://stackoverflow.com/questions/10197559/ssh-configuration-override-the-default-username

**Comment by sylirre** [comment]
For SSH this can be done using `ssh myusername@myhost` or changing `~/.ssh/config` as you said.

But it is not possible to change your actual user name under which you logged in to Termux, i.e. the one reported by `id` (like `u0_a123`). This name is generated on the fly by libc from number of parameters (device user id, runtime Linux user id) when when resolving user properties through `getpwuid()` function or similar.
